src/components/NavigationBar/index.js

```diff
@@ -327,8 +327,12 @@ class NavigationBar extends Component {
 
   handleLanguageChange = (event, index, values) => {
     localStorage.setItem('languages', event.target.value);
+    let languages = [...event.target.value];
```

Comment:
Why using spread operator here?
Can' t we just do, 
```
let languages = event.target.value;
```

Reply:
Ok @akshatnitd I agree , will make the changes.